cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControlUtils.java

```diff
@@ -82,6 +82,40 @@ public static String toDateString(long time, String dateFormat, String timeZone)
     return formatter.format(new Date(time));
   }
 
+  /**
+   * Format the duration from double to human readable string.
+   * @param durationMs Duration in milliseconds
+   * @return String representation of duration
+   */
+  public static String toDurationString(double durationMs) {
```

Comment:
Nit: For consistency with `CruiseControlUtils#toPrettyTime`, should we rename this function to `toPrettyDuration`?